pkg/wrapper/bundlewrapper.go

```diff
@@ -149,38 +148,3 @@ func (bw *BundleWrapper) AllComponents() []*bundle.Component {
 	}
 	return nil
 }
-
-// ExportAsObjects will export a Bundle as a ComponentSet and Components,
-// or a Component as unstructured Objects.
-func (bw *BundleWrapper) ExportAsObjects() ([]*unstructured.Unstructured, error) {
```

Comment:
I use this method in other tooling, I need it or an equivalent.

Reply:
Do the methods in `exporter.go` work for you?

Reply:
It might work, I think I can use ComponentsAsSingleYAML then convert that to unstructured.